clip_image.py

```diff
@@ -0,0 +1,85 @@
+from typing import Optional, Iterable, Any, List
+
+import clip
+import numpy as np
+import torch
+from PIL import Image
+from jina import Executor, DocumentArray, requests
+
+
+def _batch_generator(data: List[Any], batch_size: int):
+    for i in range(0, len(data), batch_size):
+        yield data[i:min(i + batch_size, len(data))]
+
+
+class CLIPImageEncoder(Executor):
+    """
+    Encode image into embeddings.
+
+    :param model_name: use clip.available_models() to see all available models: ['RN50', 'RN101', 'RN50x4', 'ViT-B/32']
+    :param device: device to use for encoding ['cuda', 'cpu] - if not set, the device is detected automatically
```

Comment:
missing parameter use default preprocessor

Reply:
channel axis not needdd

Reply:
added thanks